rest/taskana-rest-spring/src/test/java/pro/taskana/common/rest/TaskanaSpringBootTest.java

```diff
@@ -15,6 +17,7 @@
 /** Use this annotation to test with a spring context and a standardized configuration. */
 @Target(ElementType.TYPE)
 @Retention(RetentionPolicy.RUNTIME)
+@DirtiesContext(classMode = ClassMode.AFTER_CLASS)
```

Comment:
We talked about this briefly. What was the reason you had to mark every test as dirty? Can you document this and also state that this is just temporary?